src/browser_action/vitals.js

```diff
@@ -210,12 +219,21 @@
       // of animations or highly-dynamic content, we
       // debounce the broadcast of the metric.
       latestCLS = metric;
+      if(enableLogging) {
```

Comment:
Should each of these metrics' logs be moved to `broadcastMetricsUpdates`? That will also debounce the CLS logging to prevent console spam.

One other suggestion is to prefix the log message with something identifiable for filtering, eg `"[Web Vitals] "`

Reply:
Hi Rick, thanks for your feedback.

Yes I agree with you, at first I've added the logging within `broadcastMetricsUpdates` but then I've moved to the getMetricName method because I wanted to create one line per Metric logging.

I've moved the logging to that method, I thought about keeping the metric value rounded toFixed(2) in the first logging value and send as last log entry the whole metric object.

Maybe we can support different logging levels, such as debug (metric name + value) and verbose (metric name, value, metric object with all the entries).
WDYT?

Below the example screenshot
![Screenshot 2020-11-17 at 19 58 09](https://user-images.githubusercontent.com/329023/99434589-41caf380-290f-11eb-9e92-24d48b6c514c.png)

Reply:
> Maybe we can support different logging levels, such as debug (metric name + value) and verbose (metric name, value, metric object with all the entries).
> WDYT?

I think the way you have it here is a good first step for this PR and we can keep #77 open to revisit it with enhancements like the one you suggested.